lib/theme_check/shopify_liquid/source_index.rb

```diff
@@ -11,26 +11,49 @@
 require_relative 'source_index/return_type_entry'
 require_relative 'source_index/tag_entry'
 
+require_relative 'source_index/base_state'
+require_relative 'source_index/filter_state'
+require_relative 'source_index/object_state'
+require_relative 'source_index/tag_state'
+
```

Comment:
We believe we might centralize the require calls here: [`lib/theme_check/shopify_liquid.rb`](https://github.com/Shopify/theme-check/blob/language-server-docs-main/lib/theme_check/shopify_liquid.rb) - I've noticed this pattern after the merge of the `SourceIndex` PR 😅